release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1368,7 +1368,7 @@ The `authentication` and `openshift-apiserver` Operators now ignore the `oauth-a
 
 * Previously, when using dual-stack networking, worker node host names sometimes did not match the host name that Ironic inspected before deployment. This caused nodes to need manual approval. This has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1955114[*BZ#1955114*])
 
-* Previously, in UEFI mode, the `ironic-python-agent` created a UEFI bootloader entry after downloading the RHCOS image. When using an RHCOS image based on RHEL 8.4, the image could fail to boot using this entry. If the entry installed by Ironic was used when booting the image, the boot could fail and output a BIOS error screen. This is fixed by the `ironic-python-agent` configuring the boot entry based on a CSV file located in the image, instead of using a fixed boot entry. The image boots properly without error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966129[*BZ#1966129*])
+* Previously, in UEFI mode, the `ironic-python-agent` created a UEFI bootloader entry after downloading the RHCOS image. When using an RHCOS image based on RHEL 8.4, the image could fail to boot using this entry. If the entry installed by Ironic was used when booting the image, the boot could fail and output a BIOS error screen. This is fixed by the `ironic-python-agent` configuring the boot entry based on a CSV file located in the image, instead of using a fixed boot entry. The image boots properly without error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1972213[*BZ#1972213*])
```

Comment:
Are you sure that this BZ link needs to change?

Reply:
Yes, I got confirmation from the assignee that BZ#1966129 was incorrect (that is a clone). It needed to be updated to BZ#1972213.

Reply:
Thanks! The first bug sounded more like the issue that was being addressed here. :)